core/domain/topic_services.py

```diff
@@ -826,6 +832,8 @@ def save_topic_summary(topic_summary):
         'uncategorized_skill_count': topic_summary.uncategorized_skill_count,
         'subtopic_count': topic_summary.subtopic_count,
         'total_skill_count': topic_summary.total_skill_count,
+        'total_published_node_count':
```

Comment:
General question: how do we ensure that this summary is updated when a downstream service changes its list of explorations? The summary should automatically be recomputed when the topic itself changes (e.g. which stories are in the list), but changes to the stories themselves won't affect the topic summary. Won't that mean this will get out of date?

Reply:
Answered via email. Hoping to run a job beforehand.